src/HeapDump/GCHeapDumper.cs

```diff
@@ -171,6 +171,12 @@ private CollectionMetadata CaptureLiveHeapDump(int processID)
 
         m_log.WriteLine("Process Has DotNet: {0} Has JScript: {1} Has ClrDll: {2} HasMrt {3} HasCoreClr {4}", hasDotNet, hasJScript, hasClrDll, hasMrt, hasCoreClr);
 
+        if (!hasDotNet && !hasJScript && !hasMrt && !hasCoreClr)
+        {
+            m_log.WriteLine("No supported runtime type detected, going to assume native AOT.");
```

Comment:
Is there another way to detect that we're dealing with Native AOT?  I suppose that trying this as a last resort is OK if necessary, since we can currently detect the other cases.